docontrol/README.md

```diff
@@ -0,0 +1,71 @@
+# DoControl
+
+## Overview
+
+This integration allows [DoControl](https://www.docontrol.io/) customers to forward their DoControl-related logs and events to Datadog through automated security workflows.
+
+## Setup
+
+To set up this integration, you must have an active [DoControl account](https://www.docontrol.io/demo). You must also have proper admin permissions in Datadog.
```

Comment:
```suggestion
To set up this integration, you must have an active [DoControl account][10]. You must also have proper [admin permissions][11] in Datadog.
```

Reply:
Apologies for missing the link format issue on these in my first pass. Using reference links is helpful for the translation process, as mentioned in our contributing guidelines.
I've additionally added a link to the Datadog permissions section here. It's worth pointing out that `admin` permissions have been deprecated and replaced with more granular permission sets, as noted in the linked document. As such, it may be worthwhile to define which specific permission sets are required, but I'll leave that up to you.